public/es6.html

```diff
@@ -31,14 +31,14 @@
     <div class="description">
 
       <p>io.js is built against modern versions of <a href="https://code.google.com/p/v8/">V8</a>. By keeping up-to-date with the latest releases of this engine we ensure new features from the <a href="http://www.ecma-international.org/publications/standards/Ecma-262.htm">JavaScript ECMA-262 specification</a> are brought to io.js developers in a timely manner, as well as continued performance and stability improvements.</p>
-      <p>Version 1.0.3 of io.js ships with V8 4.1.0.7, which includes ES6 features well beyond version 3.26.33 that will be shipped with joyent/node@0.12.x.</p>
+      <p>Version 1.0.3 of io.js ships with V8 4.1.0.7, which includes ES6 features well beyond version 3.28.73 that will be shipped with joyent/node@0.12.x.</p>
     </div>
 
     <div class="faq-item">
 
       <h2 class="faq-title">No more --harmony flag</h2>
       <div class="faq-body">
-        <p>On joyent/node@0.12.x (V8 3.26), the <code>--harmony</code> runtime flag enabled all <strong>completed</strong>, <strong>staged</strong> and <strong>in progress</strong> ES6 features together, in bulk (with the exception of nonstandard/non-harmonious semantics for <code>typeof</code> which were hidden under <code>--harmony-typeof</code>).  This meant that some really buggy or even broken features like <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy">proxies</a> were just as readily available for developers as <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*">generators</a>, which had very little or even no known-issues. As such, it was best practice to either enable only certain features by using specific runtime harmony feature flags (e.g. <code>--harmony-generators</code>), or simply enable all of them and then use a restricted subset.</p>
+        <p>On joyent/node@0.12.x (V8 3.28), the <code>--harmony</code> runtime flag enabled all <strong>completed</strong>, <strong>staged</strong> and <strong>in progress</strong> ES6 features together, in bulk (with the exception of <code>proxies</code> which were hidden under <code>--harmony-proxies</code>). This meant that some really buggy or even broken features like <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions">arrow functions</a> were just as readily available for developers as <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*">generators</a>, which had very little or even no known-issues. As such, it was best practice to either enable only certain features by using specific runtime harmony feature flags (e.g. <code>--harmony-generators</code>), or simply enable all of them and then use a restricted subset.</p>
```

Comment:
Correct me if I'm wrong, but I don't think this applies to the V8 version that ships with node 0.12 (3.28) anymore.